metadata-models/src/main/pegasus/com/linkedin/glossary/GlossaryTermInfo.pdl

```diff
@@ -0,0 +1,36 @@
+namespace com.linkedin.glossary
+
+import com.linkedin.common.Uri
+import com.linkedin.common.GlossaryNodeUrn
+import com.linkedin.common.TermSource
+
+/**
+ * Properties associated with a GlossaryTerm
+ */
+record GlossaryTermInfo {
```

Comment:
Can we add a reference to the GlossaryTerms's parent node here as well? Same reason as above: this way we would be able to access the parent node without having to rely on string parsing of the Term's Urn.

Reply:
[same as above]

Reply:
added parentNode